app/views/idv/come_back_later/show.html.erb

```diff
@@ -17,18 +17,25 @@
 
 <%= render PageHeadingComponent.new(class: 'text-center').with_content(t('idv.titles.come_back_later')) %>
 
+<div class="text-center">
+  <%= sanitize t('idv.messages.come_back_later_html', app_name: APP_NAME), tags: %w[p strong br] %>
```

Comment:
With the `_html` suffix, we shouldn't need to deal with escaping of the HTML. Or does this exist to try to restrict what HTML might exist? We don't do that anywhere else unless we're dealing with "user-generated" content like [provider banner messages](https://github.com/18F/identity-idp/blob/fd14459a6af21cd419c2397e34dd37b1dda5d4e8/app/views/shared/_sp_alert.html.erb#L4), so we could probably leave it out.

```suggestion
  <%= t('idv.messages.come_back_later_html', app_name: APP_NAME) %>
```